app/poi_api/poi_rest_handler.py

```diff
@@ -0,0 +1,33 @@
+from .poi_service import POI_Service
+from flask import jsonify
+from .poi_errors import POINotFoundError, InvalidPOISuggestionError
+
+poi_service_class = POI_Service()
+
+# TODO Exception handling
+# TODO type hints
+def get_all_POI():
+    try:
+        list_all_poi = poi_service_class.get_all_POI()
+        return [poi.to_dict() for poi in list_all_poi], 200
+    except Exception as e:
+        return jsonify({"success": False, "message": str(e), "error_code": 400}), 400

```

Comment:
You don't have to jsonify the return values, why do you have to jsonify the errors?